src/create-model.jl

```diff
@@ -266,6 +267,11 @@ function add_expression_terms_intra_rp_constraints!(
 
     for case in cases
         df_cons[!, case.col_name] .= JuMP.AffExpr(0.0)
+        conditions_to_add_min_outgoing_flow_duration =
+            add_min_outgoing_flow_duration && case.col_name == :outgoing_flow
+        if conditions_to_add_min_outgoing_flow_duration
+            df_cons[!, :min_outgoing_flow_duration] .= 1
```

Comment:
```suggestion
            df_cons[!, :min_outgoing_flow_duration] .= 1.0
```

Reply:
Nevermind, it's supposed to be integer, right?

Reply:
correct, it is integer 😄

Reply:
👍